app/src/main/java/com/github/polybooks/database/BookDatabase.kt

```diff
@@ -19,6 +19,11 @@ interface BookDatabase {
      * */
     fun queryBooks() : BookQuery
 
+    /**
+     * Adds/Modifies a book in the database
+     * */
+    fun addBook(book: Book): CompletableFuture<Unit>
```

Comment:
What about a function to specifically modify the rating of the book (since this is what you want to achieve)?

Reply:
Nevermind, I will probably have a ```addBook()``` function when I'll do the book cache. It's not perfect, but we can keep it.